test/extractor/CaretValueRuleExtractor.test.ts

```diff
@@ -110,36 +110,18 @@ describe('CaretValueRuleExtractor', () => {
 
     it('should not extract caret rules when an array is exactly the same as the inherited array', () => {
       const sd = cloneDeep(looseSD);
-      sd.mapping = [
+      sd.context = [
```

Comment:
Oops.  Sorry I didn't have the foresight to realize these tests would break as soon as we added mapping support!